python/pynq/pl.py

```diff
@@ -152,6 +152,165 @@ def _get_gpio(tcl_name):
 
     return result
 
+class _InterruptMap:
```

Comment:
Wouldn't that violate the PEP-8 naming convention? I've addressed all other comments in a review commit.